### ISSUE TITLE
module 'pysal.explore.spaghetti' has no attribute 'element_as_gdf'

### ISSUE THREAD

**Issue by TransUMD** [pysal/spaghetti#323]
**Describe the bug**
module 'pysal.explore.spaghetti' has no attribute 'element_as_gdf'

**To Reproduce**
Running the example Spaghetti_Pointpatterns_Empirical.ipynb

**Comment by jGaboardi** [comment]
@TransUMD, thanks for the report! I should be able to look into this in the next several days.

**Comment by TransUMD** [comment]
Thanks! One more information is that I install Pusan directly via pip install pysal. The raw code here have this function but I am not sure whether my version is right.

Thanks.

**Comment by jGaboardi** [comment]
Do you mean `pysal`? (Maybe Pusan because of autocorrect). 


```python
import pysal
import spaghetti
print(pysal.__version__)
print(spaghetti.__version__)
```

Also, keep in mind that `geopandas` must be installed to be able to use [`element_as_gdf`](https://github.com/pysal/spaghetti/blob/cf7908bae2d6686ec7d72b9e8ae0e92d035bd2a3/spaghetti/network.py#L2303).

**Comment by TransUMD** [comment]
Hello,
Thanks for the prompt reply.

I do have geopandas installed in my computer.

The version of these are.

pysal: 2.0.0
spaghetii: 1.3.1
geopandas: 0.5.1

Thanks

**Comment by TransUMD** [comment]
It also happens to me when running:

counts = ntw.count_per_link(ntw.pointpatterns['crimes'].obs_to_arc,
                            graph=False)

I got:

'Network' object has no attribute 'count_per_link'

**Comment by jGaboardi** [comment]
It sounds like you may have corrupted installation of `spaghetti`. Other than trying a fresh install of `spaghetti`, I am not sure how to troubleshoot this. 

@sjsrey @ljwolf @weikang9009 Any suggestions here?

**Comment by TransUMD** [comment]
Thank you. I tried to install spaghetti separately and it works now by just import spaghetti.

Not sure why it does not work with: import pysal.explore.spaghetti

**Comment by jGaboardi** [comment]
@TransUMD, this may very well be an issue with how we are bundling `spaghetti` into the full `pysal` metapackage. Thank you for bringing it to our attention! Let's keep this issue open here while I try to dig a bit deeper.

**Comment by TransUMD** [comment]
Thank you!

**Comment by weikang9009** [comment]
@TransUMD @jGaboardi Maybe try installing the most recent version of pysal? ([pysal 2.1.0](https://pypi.org/project/pysal/))

`pip install -U pysal`

**Comment by jGaboardi** [comment]
@weikang9009 good catch! I had missed that @TransUMD was working from `pysal v2.0.0`!

@TransUMD `spaghetti v1.1.0` is included with `pysal v2.0.0`. This version of `spaghetti` does not include `Network.element_as_gdf()` and `Network.count_per_link()` was renamed from `Network.count_per_edge()`.

**Comment by TransUMD** [comment]
@weikang9009 Thanks. I re-install pysal and the problem is solved. Thank you all for the response!
